.github/workflows/azure-static-web-apps-AZ.yml

```diff
@@ -24,3 +26,5 @@ jobs:
           app_location: '/' # App source code path relative to repository root
           app_build_command: 'npm run build:test'
           output_location: 'build' # Built app content directory, relative to app_location - optional
+        env: # Add environment variables here
+          REACT_APP_STAGING_ENV: ${{ github.event_name == 'pull_request' && 'true' || 'false' }}
```

Comment:
why do we need `&& 'true' || 'false'` here?

Reply:
That is a fair point. The solution was derived from more general one:
`'pull_request' && 'staging-value' || 'production-value'`
so I wasn't really thinking about it